include/ocpp/v201/charge_point.hpp

```diff
@@ -38,6 +40,8 @@ struct Callbacks {
     std::function<void(const int32_t evse_id, const ReasonEnum& stop_reason)> stop_transaction_callback;
     std::function<void(const int32_t evse_id)> pause_charging_callback;
     std::function<void(const ChangeAvailabilityRequest& request)> change_availability_callback;
+    std::function<void(const RequestStartTransactionRequest& request)> remote_start_transaction_callback;
+    std::function<void(const CiString<36>& transaction_id)> remote_stop_transaction_callback;
```

Comment:
There already is a stop_transaction_callback, with parameters evse_id and stop_reason.

Reply:
> There already is a stop_transaction_callback, with parameters evse_id and stop_reason.

Yes I would reuse this here. Maybe we need to change the parameters of this function so that the app that uses libocpp can properly report back to libocpp, but we can also add this later